source/change-streams/change-streams.rst

```diff
@@ -307,6 +307,26 @@ If an aggregate command with a ``$changeStream`` stage completes successfully, t
      * @since 4.7.0
      */
     truncatedArrays: Array<Document>;
+
+    /**
+     * A document containing a map that associates an update path to an array containing the path components used in the update document.  This data
+     * can be used in combination with the other fields in an `UpdateDescription` to determine the actual path in the document that was updated. This is 
+     * necessary in cases where a key contains a dot-separated strings (i.e., `{ "a.b": "c" }`) or a document contains a numeric literal string key
+     * (i.e., `{ "a": { "0": "a" } }`.  Note that in this scenario, the numeric key can't be the top level key, because `{ "0": "a" }` is not ambiguous - 
```

Comment:
```suggestion
     * i.e., `{ "a": { "0": "a" } }`. Note that in this scenario, the numeric key can't be the top level key, because `{ "0": "a" }` is not ambiguous - 
```